tests/integration/e2e/test_universal_e2e.py

```diff
@@ -0,0 +1,136 @@
+import math
+from datetime import datetime, timedelta
+from typing import Optional
+
+import pandas as pd
+from pytz import utc
+
+from feast import FeatureStore, FeatureView
+from tests.integration.feature_repos.test_repo_configuration import (
+    Environment,
+    parametrize_e2e_test,
+)
+from tests.integration.feature_repos.universal.entities import driver
+from tests.integration.feature_repos.universal.feature_views import driver_feature_view
+
+
+@parametrize_e2e_test
```

Comment:
A pretty big pain point for me is that I can't see how this test is being parameterized. What are all the test cases we are going to use to test e2e consistency? It seems hidden from me as a test author. Ideally somebody would just need to look at this single file to see how a test would be parameterized. It's highly unlikely that all the different test files would use the exact same set of parameters (which could benefit from generalization/reuse), so it seems more natural to put the cases here.